src/System.Security.Cryptography.X509Certificates/tests/FindTests.cs

```diff
@@ -735,6 +771,15 @@ public static void TestBySubjectKeyIdentifier_ExtensionPresent(string subjectKey
             RunSingleMatchTest_MsCer(X509FindType.FindBySubjectKeyIdentifier, subjectKeyIdentifier);
         }
 
+        // Should ignore Left-to-right mark \u200E
+        [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "31463 is not fixed in NetFX")]
+        [InlineData(_leftToRightMark + "59 71 A6 5A 33 4D DA 98 07 80 FF 84 1E BE 87 F9 72 32 41 F2")]
+        public static void TestBySubjectKeyIdentifier_ExtensionPresentWithLTM(string subjectKeyIdentifier)
```

Comment:
The "trailing nybble" cases from the next test should be copied to here (with the LTR mark prefixed)